RELEASE.md

```diff
@@ -1,14 +1,14 @@
 # Release steps
 
 1. Ensure master passes CI tests
-2. `npm run gendocs`
-3. Bump version, create tags, push, and release:
+1. Bump version, create tags, push, and release:
   - `$ npm run <release:major|release:minor|release:patch>`
   - `major` - breaking API changes
   - `minor` - backwards-compatible features
   - `patch` - backwards-compatible bug fixes
-4. Update CHANGELOG.md
+1. Update `CHANGELOG.md`
   - `$ npm run changelog`
+  - Manually verify that the changelog makes sense
   - `$ git push`
-5. Generate the documentup website by visiting
-  [http://documentup.com/shelljs/shelljs/__recompile] in your browser
+1. Generate the documentup website by visiting
```

Comment:
Sure. For what it's worth, the number can actually be anything, it doesn't have to increment. Here are the [official docs](https://help.github.com/articles/basic-writing-and-formatting-syntax/#lists) and here is a guide which [demonstrates that](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists).

I've seen the 1-only style preferred other places. I suppose this might likely be read in the terminal, so the real numbers might be more helpful.